modules/apps/frontend-token/frontend-token-definition-impl/src/main/java/com/liferay/frontend/token/definition/internal/TokenDefinitionImpl.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.token.definition.internal;
+
+import com.liferay.frontend.token.definition.TokenDefinition;
+
+/**
+ * @author Iván Zaera
+ */
+public class TokenDefinitionImpl implements TokenDefinition {
+
+	public TokenDefinitionImpl(String rawDefinition) {
```

Comment:
I might be wrong, but maybe Brian preferes `rawTokenDefinition`?

Reply:
If we rename the method, yes.